apps/studio/src/lib/db/clients/postgresql.ts

```diff
@@ -1347,6 +1348,15 @@ export async function truncateAllTables(conn: Conn, schema: string) {
   });
 }
 
+export async function dropElement (conn: Conn, elementName: string, typeOfElement: DatabaseElement, schema: string): Promise<void> {
```

Comment:
good stuff, but see above regarding wrapLiteral

Reply:
What if there's no schema specified?

Reply:
I was under the impression schema had to be passed in with postgres (at least that's how I've always done it but I don't leave much to chance)? If not, I can add an if in there to get different sql generated.

Reply:
So probably the app always passes schema, but maybe not. It's possible I'm not consistent about this either, but I try to be. Everything will work without schema, it will just default to `public`